source/YugiAcademy/assertions/assertions.py

```diff
@@ -0,0 +1,16 @@
+from .instance_creation_failed import InstanceCreationFailed
+
+
+def enforce_not_blank(string, name):
+    if not string.strip():
+        raise InstanceCreationFailed(f'{name} must not be blank.')
+
+
+def enforce_must_be_between(lower_bound, number_to_compare, upper_bound, number_name):
+    if not (lower_bound <= number_to_compare <= upper_bound):
+        raise InstanceCreationFailed(f'{number_name} must be between {lower_bound} and {upper_bound}.')
+
+
+def enforce_must_be_positive(number, number_name):
```

Comment:
Greater or equal than 0! Positive not includes 0.
![image](https://user-images.githubusercontent.com/29506971/191015479-6ba4a8df-be54-471a-89cc-21757fedfc7b.png)

Reply:
It seems that all my life I lived a lie. I always though that zero was a positive number. Ok, I'm going to rename this method. I was confused with the strictly positive concept.